projects/pyqt.cmake

```diff
@@ -11,21 +11,36 @@ if(BUILD_OS_WINDOWS)
         "CFLAGS+=${CMAKE_C_FLAGS}"
         "CXXFLAGS+=${CMAKE_CXX_FLAGS}"
     )
+# HAS THIS BEEN REMOVED BEFORE?
```

Comment:
Ok, I was merging my old work into this PR and wondered why this part of the code was lost.